LICENSE.txt

```diff
@@ -1,9 +1,18 @@
 Copyright (c) 2014-2019 The Khronos Group Inc.
 
-Khronos licenses this file to you under the Creative Commons Attribution 4.0
-International (CC BY 4.0) License (the "License"); you may not use this file
-except in compliance with the License.  You may obtain a copy of the License
-at https://creativecommons.org/licenses/by/4.0/
+The files in, and generated output documents from this SYCL-Docs project are
+under a mix of copyright and license statements. Refer to the individual files
+for specific information. As a general guideline:
+
+- the LaTeX sources for the SYCL Specifications and other documention
+  such are the SYCL code samples
+  are under the Creative Commons Attribution 4.0 International (CC BY 4.0)
+  license.  Details of the license are at:
+  https://creativecommons.org/licenses/by/4.0/
+
+- the source files for building the specification, such as Makefiles
```

Comment:
Maybe also say "headers" as the .h files are Apache 2.0? Otherwise, this looks good to me. Thanks for working through the unpleasant license/legal issues!

Reply:
Done.
Fortunately there was some LaTeX developments required to add a little bit of fun to this administrative chore... :-)